httpcore5/src/main/java/org/apache/hc/core5/http/message/MessageSupport.java

```diff
@@ -151,6 +152,11 @@ public static void addContentEncodingHeader(final HttpMessage message, final Ent
     }
 
     public static void addTrailerHeader(final HttpMessage message, final EntityDetails entity) {
+        // Ensure using chunked transfer encoding
```

Comment:
@arturobernalg Please, do not. This is very wrong on many levels. First off `Transfer-Encoding` value is a not a single token, it is a comma delimited sequence, like `this, that, chunk. Secondly off, this is absolutely the wrong place to enforce the check. Thirdly it is not even necessary because the `*tContent` protocol interceptors already take care of.